app/about/faq/text.tsx

```diff
@@ -12,7 +12,7 @@ export enum FAQPageOptionId {
 export const faqPageText: Record<FAQPageOptionId, {label: string, content: JSX.Element}> = {
   [FAQPageOptionId.WHERE_DOES_SEROTRACKERS_DATA_COME_FROM]: {
     label: "Where does SeroTracker's data come from?",
-    content: <p className='inline'> We collect all seroprevalence data from published peer reviewed research articles, preprints, reports, and media (unpublished grey literature). The source of any individual estimate can be viewed by clicking on the estimate on the map or by referencing the Source column in the table in the dashboard or the url column in a csv downloaded from the table. <Link className="inline text-link underline" target="_blank" href="/pathogen/arbovirus/wip#table">This is a link to the ArboTracker map</Link> and <Link className="inline text-link underline" target="_blank" href="/pathogen/arbovirus/wip#table">this is a link to the ArboTracker data table</Link>.</p>
+    content: <p className='inline'> We collect all seroprevalence data from published peer reviewed research articles, preprints, reports, and media (unpublished grey literature). The source of any individual estimate can be viewed by clicking on the estimate on the map or by referencing the Source column in the table in the dashboard or the url column in a csv downloaded from the table. <Link className="inline text-link underline" target="_blank" href="/pathogen/arbovirus/dashboard#table">This is a link to the ArboTracker map</Link> and <Link className="inline text-link underline" target="_blank" href="/pathogen/arbovirus/dashboard#table">this is a link to the ArboTracker data table</Link>.</p>
```

Comment:
> Also, Can we make it so that the visualization section (the part of the page with all the visualizations is not limited to h-screen and instead just takes the size of all viz. That will avoid this from happening:

Yes we can! Resolved in cf40742b1ddf5d71ac66b5160e1f32fc4257235f!